packages/taquito-local-forging/test/taquito-local-forging.spec.ts

```diff
@@ -644,6 +661,195 @@ const cases: TestCase[] = [
       ],
     },
   },
+  {
+    name: 'Origination with sapling 1',
+    operation: {
+      branch: 'BLzyjjHKEKMULtvkpSHxuZxx6ei6fpntH2BTkYZiLgs8zLVstvX',
+      contents: [
+        {
+          kind: 'origination',
+          counter: '1',
+          source: 'tz1QZ6KY7d3BuZDT1d19dUxoQrtFPN2QJ3hn',
+          fee: '10000',
+          gas_limit: '10',
+          storage_limit: '10',
+          balance: '0',
+          script: {
+            code: rpcContractResponse.script.code,
+            storage: [],
+          },
+        },
+      ],
+    },
+  },
+  {
+    name: 'Origination with sapling 2',
```

Comment:
These tests names should describe hot they are different from each other. Self-descriptive :)